runtime/flangrti/CMakeLists.txt

```diff
@@ -88,14 +88,14 @@ if (NOT DEFINED LIBOMP_EXPORT_DIR)
     FLANG_LIBOMP
     NAMES omp libomp
     HINTS ${CMAKE_BINARY_DIR}/lib)
-  target_link_libraries(flangrti_shared PRIVATE ${FLANG_LIBOMP})
+  target_link_libraries(flangrti_shared PUBLIC ${FLANG_LIBOMP})
```

Comment:
I have some linking issue on Windows because of some unresolved symbols from libpgmath. It seems it fixes this issue.  What is the differences if we change it PUBLIC or leave it "empty"? What is the default setting?

Reply:
See https://cmake.org/cmake/help/git-master/policy/CMP0023.html.

Making it PUBLIC is the correct thing here as we can't mix `target_link_libraries` with and without `PUBIC/PRIVATE/LINK_INTERFACE`. We want `libomp` to be part of the link interface, so that dependent libraries like `libflang` also links to `libomp`.